### PR TITLE
Link to publication.

### PR DESCRIPTION
**Pull request by yeban**

### CODE REVIEW

File: lib/genevalidatorapp.rb [hunk 1]
```diff
@@ -75,8 +75,9 @@ def on_stop
       puts
       puts '** Thank you for using GeneValidatorApp :).'
       puts '   Please cite: '
-      puts '        Dragan M., Moghul M.I., Priyam A., Wurm Y (in prep).'
-      puts '        GeneValidator: identify problematic gene predictions.'
+      puts '        Dragan M, Moghul MI, Priyam A, Bustos C, Wurm Y. 2015.'
```
Comment (IsmailM) [inline lib/genevalidatorapp.rb]: 2016, not 2015 :)

File: public/GeneValidator/loading.html [hunk 2]
```diff
@@ -9,4 +9,4 @@ <h4 style="margin-bottom:3em">Identify Problems with Protein-Coding Gene Predict
...
\ No newline at end of file
+<footer><div class="container center-block"><p class="text-muted text-center">Please cite: <a href="http://bioinformatics.oxfordjournals.org/content/early/2016/01/18/bioinformatics.btw015">Dragan M<sup>&Dagger;</sup>, Moghul MI<sup>&Dagger;</sup>, Priyam A, Bustos C &amp; Wurm Y. 2015. GeneValidator: identify problems with protein-coding gene predictions". <em>Bioinformatics</em>, doi: 10.1093/bioinformatics/btw015</a>.<br/>Developed at <a href="https://wurmlab.github.io" target="_blank">Wurm Lab</a>, <a href="http://www.sbcs.qmul.ac.uk" target="_blank">QMUL</a> with funding by <a href="http://www.bbsrc.ac.uk/home/home.aspx" target="_blank">BBSRC</a> and <a href="https://www.google-melange.com/gsoc/homepage/google/gsoc2013" target="_blank">Google Summer of Code 2013</a><br/>This page was created by <a href="https://github.com/wurmlab/genevalidator" target="_blank" >GeneValidator</a> v1.6.5</p></div></footer></body></html>
```
Comment (IsmailM) [inline public/GeneValidator/loading.html]: again here, 2016

File: views/layout.slim [hunk 3]
```diff
@@ -55,10 +55,9 @@ html lang="en"
             i.fa.fa-spinner.fa-5x.fa-spin
     footer#footer
       p.text-muted.text-center
-        | Please cite "Dragan M<sup>&Dagger;</sup>, Moghul MI<sup>&Dagger;</sup>, Priyam A, Bustos C &amp; Wurm Y 
-        em
-          | (in prep) 
-        | GeneValidator: identify problematic gene predictions"
+        | Please cite: 
+        <a href="http://bioinformatics.oxfordjournals.org/content/early/2016/01/18/bioinformatics.btw015">Dragan M<sup>&Dagger;</sup>, Moghul MI<sup>&Dagger;</sup>, Priyam A, Bustos C &amp; Wurm Y. 2015. GeneValidator: identify problematic gene predictions.
```
Comment (IsmailM) [inline views/layout.slim]: 2016

**Comment by yeban** [comment]
@IsmailM Fixed!

**Pull request merged**